data/handler/handler.go

```diff
@@ -0,0 +1,106 @@
+package handler
+
+import (
+	"encoding/csv"
+	"io"
+	"log"
+	"os"
+	"strconv"
+
+	"github.com/wdonet/golang-bootcamp-2020/domain/model"
+	"github.com/wdonet/golang-bootcamp-2020/utilities"
+)
+
+const filename string = "./data/todos.csv"
+
+// GetTodosFromFile get data from files as Todos
+func GetTodosFromFile() []*model.Todo {
+	csvfile, err := os.Open(filename)
+	if err != nil {
+		log.Println("Unable to open for read CSV file!", err)
+		return nil
```

Comment:
A common practice for error handling in Go is to almost always return the error. In this case, your function can return an empty To-do list alongside the error.